lighthouse-core/test/gather/computed/main-thread-tasks-test.js

```diff
@@ -14,15 +14,18 @@ const TracingProcessor = require('../../../lib/traces/tracing-processor.js');
 const assert = require('assert');
 
 describe('MainResource computed artifact', () => {
-  const args = {data: {}};
+  const pid = 1;
+  const tid = 2;
+  const frameId = 'BLAH';
+  const args = {data: {}, frame: frameId};
```

Comment:
this test implicitly relied on many fields (tid, pid, ...) being the same (in this case, undefined), so that events would be collected (see `trace-of-tab.js` `mainThreadEvents` for one example). Also, since an error is thrown on a non-truthy id value, these had to be given some positive number.